src/main/java/seedu/financialplanner/utils/Ui.java

```diff
@@ -61,9 +71,12 @@ public void printWatchListHeader() {
     public void printStocksInfo(WatchList watchList) {
         for (Stock stock: watchList.getStocks()) {
             String symbol = StringUtils.rightPad(stock.getSymbol(), 10);
-            String price = StringUtils.rightPad(stock.getPrice(), 10);
-            String name = StringUtils.rightPad((String) stock.getStockName(), 10);
-            System.out.println(symbol + price + name);
+            String market = StringUtils.rightPad(stock.getExchange(), 10);
+            String price = YELLOW + StringUtils.rightPad(stock.getPrice(), 10) + RESET;
+            String dayHigh = GREEN + StringUtils.rightPad(stock.getDayHigh(), 15) + RESET;
+            String dayLow = RED + StringUtils.rightPad(stock.getDayLow(), 14) + RESET;
+            String name = StringUtils.rightPad(stock.getStockName(), 10);
+            System.out.println(symbol + market + price + dayHigh + dayLow + name);
```

Comment:
good use of creating new string variables and combining them in a single system output